### PR TITLE
get a start on multiple locations

### PR DESCRIPTION
**Pull request by briaguya**
resolves https://github.com/cityofaustin/techstack/issues/3127

### CODE REVIEW

File: src/js/queries/informationPageFragment.js [hunk 1]
```diff
@@ -51,6 +51,15 @@ const informationPageFragment = `
           contact {
             name
             email
+            phoneNumber {
```
Comment (easherma) [inline src/js/queries/informationPageFragment.js]: Why not use 
https://github.com/cityofaustin/janis/blob/d577f67d0c0f12c810aedb18a2bf56ec55d62532/src/js/queries/phoneFragment.js ?

File: src/components/Contact/ContactDetails.js [hunk 2]
```diff
@@ -20,34 +20,45 @@ import {
...
+  <React.Fragment>
     {/* We want to keep this component easy to read and streamlined, so we are
       avoiding complicated functions in this return.
       Consider breaking into another component and calling that here*/}
-    {email && <Email email={email} />}
+    <h3 className="coa-ContactDetails__name">{name}</h3>
     {location && <Address location={location} />}
-    {phoneNumber && <PhonesList phoneNumbers={phoneNumber} />}
     {hours && !!hours.length && <Hours hours={hours} />}
+    {phoneNumber && <PhonesList phoneNumbers={phoneNumber} />}
```
Comment (easherma) [inline src/components/Contact/ContactDetails.js]: did the order get changed in the design?

File: src/components/Contact/ContactDetails.js [hunk 3]
```diff
@@ -20,34 +20,45 @@ import {
...
-    phone: phonePropTypes,
-  }).isRequired,
+const ContactDetails = ({ contacts, intl }) => {
+  debugger;
+  return (
+    <div className="coa-ContactDetails">
+      <SectionHeader isSerif={true}>
+        {intl.formatMessage(i18n.questionsTitle)}
+      </SectionHeader>
+      {contacts.map(contact => (
```
Comment (easherma) [inline src/components/Contact/ContactDetails.js]: Might be good to put a comment here pointing out where the contacts get mapped from, took me awhile to find the difference

File: src/components/Contact/ContactDetails.js [hunk 4]
```diff
@@ -20,34 +20,45 @@ import {
...
 );
 
-ContactDetails.propTypes = {
-  contact: PropTypes.shape({
-    location: addressPropTypes,
-    email: emailPropTypes,
-    hours: hoursPropTypes,
-    phone: phonePropTypes,
-  }).isRequired,
+const ContactDetails = ({ contacts, intl }) => {
```
Comment (easherma) [inline src/components/Contact/ContactDetails.js]: Some sort of indication of it being plural consistent with the singular might be good. 
ContactDetail ---> ContactDetails
SingularContactDetails ---> AllContactDetails

Or something like that

File: src/components/Contact/ContactDetails.js [hunk 5]
```diff
@@ -20,34 +20,45 @@ import {
...
+        {intl.formatMessage(i18n.questionsTitle)}
+      </SectionHeader>
+      {contacts.map(contact => (
+        <SingularContactDetails contact={contact} />
+      ))}
+    </div>
+  );
 };
 
+// ContactDetails.propTypes = {
```
Comment (easherma) [inline src/components/Contact/ContactDetails.js]: is this some outstanding tech debt to make these PropTypes work?

File: src/components/Contact/ContactDetails.js (hunk 2 above)
Comment (briaguya) [reply src/components/Contact/ContactDetails.js]: yes

File: src/js/queries/informationPageFragment.js (hunk 1 above)
Comment (briaguya) [reply src/js/queries/informationPageFragment.js]: i copied from guidepage, I've updated the PR so both info page and guide page use the fragment now

File: src/components/Contact/ContactDetails.js (hunk 3 above)
Comment (briaguya) [reply src/components/Contact/ContactDetails.js]: did some clean up, I think it reads more clearly now

File: src/components/Contact/ContactDetails.js (hunk 4 above)
Comment (briaguya) [reply src/components/Contact/ContactDetails.js]: did some renaming, I think it's a bit clearer now

File: src/components/Contact/ContactDetails.js (hunk 5 above)
Comment (briaguya) [reply src/components/Contact/ContactDetails.js]: Got it uncommented

**Pull request merged**